tools/VscExtension/package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "DocFXPreview",
-  "displayName": "DocFXPreview",
-  "description": "Preview Html and token tree generated by DocFX in Visual Studio Code",
-  "version": "0.0.24",
+  "name": "DocFX",
+  "displayName": "DocFX",
+  "description": "An amazing tool to make community users using and contributing to DocFX easier and more convenient, the main features include previewing DocFX Flavored Markdown with different templates, generating token tree of markdown file",
```

Comment:
maybe we could remove it to avoid confusion. DFM preview is in embeded style while docfx preview is in users style

---
In reply to: [131841135](https://github.com/dotnet/docfx/pull/1936#discussion_r131841135) [](ancestors = 131841135,131837142)